README.md

```diff
@@ -32,4 +32,12 @@ environment like so:
 pipenv shell
 ```
 
-and then commands like `pycodestyle` and `pytest` can be run like normal.
\ No newline at end of file
+and then commands like `pycodestyle` and `pytest` can be run like normal.
+
+Additionally, we use [Travis CI](https://travis-ci.org/ubclaunchpad/rocket2.0) as
+a CI system. To run the same checks locally, please install our git hooks:
```

Comment:
In addition to installer git hooks, can also mention that user/dev can run the scripts in the folder to achieve the same effect, without committing/merging.